package.json

```diff
@@ -68,6 +68,7 @@
     "jshint": "=2.9.3",
     "mocha": "=3.1.0",
     "moment": "=2.15.1",
+    "sinon": "^1.17.7",
```

Comment:
Please fixate version like other dependenices. Not sure if `=` best approach, but at least provides some assurance of no unexpected surprises.